test/integration/ioredis/ioredis-apl.test.ts

```diff
@@ -0,0 +1,95 @@
+import Redis from "ioredis";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+
+import { IORedisAPL } from "../../../src/APL/ioredis";
+
+// These tests require a running Redis instance
+// Run with: INTEGRATION=1 pnpm test
+const runIntegrationTests =
+  process.env.INTEGRATION === "1" && process.env.SENTINEL_INTEGRATION !== "1";
+const testFn = runIntegrationTests ? describe : describe.skip;
+
+testFn("IORedis APL Integration", () => {
+  const client = new Redis({ port: 6379, host: "localhost" });
```

Comment:
Should it really hardcode address? Probably should be declared in .env or some other shared config that will be commented enough to understand where it comes from

Also I think it's quite messy with this `INTEGRATION=1`, I think we should create dedicated projects:
- units
- integration-ioredis
- integration-redis

So each is scoped